src/components/06_wrappers/Default/Default.js

```diff
@@ -138,15 +153,26 @@ Default.propTypes = {
     }),
   ).isRequired,
   requestMenu: func.isRequired,
+  filterMenu: func.isRequired,
+  filterString: string.isRequired,
 };
 
 Default.defaultProps = {
   message: null,
 };
 
-const mapStateToProps = state => ({
-  message: state.application.message || null,
-  menuLinks: state.application.menuLinks || {},
-});
+const mapStateToProps = state => {
+  return {
+    message: state.application.message || null,
+    menuLinks:
+      state.application.filteredMenuLinks.length > 0
```

Comment:
For me it seems like a bit strange UX that we show all results if none is found with the given string. How about we change this to just check if `filterString` exists?

Reply:
Oh good point! :+1: